src/pages/docs/git.md

```diff
@@ -0,0 +1,287 @@
+
+To create your own branch, run the command
+
+```
+git switch -c <YOUR INITIALS>/<FEATURE>
+```
+
+Your initials correspond to the first letter of your first and last name respectively, in lowercase. So, for example, if my name is John Doe, that would be `jd`. The feature corresponds to what you are working with. This should be very short (a couple of words maximum) and descriptive. Words should be separated by a hyphen (`-`). For example, if John Doe was working on implementing a new team page, that could be `new-team-page`. Putting it all together, the command would be
+
+```
+git switch -c jd/new-team-page
+```
+
+
+With that done, open your favourite text editor. We recomend VS Code, but you can pick your own.
+
+### I have finished my work, now what?
+
+Now that you are done (for now), you need to commit your changes to the remote repository (on GitHub). First check which files you have changed by running
+
+```
+git status
+```
+
+Of these files, check which ones you want to commit and run
+
+```
+git add <files>
+```
+
+It should be noted you can add multiple files at once, including folders, patterns, .etc, like the following example
+
+```
+git add pages/ components/*.tsx package.json
+```
+
+You can check it was added successfully by rerunning `git status`. Now commit your changes by running
+
+```
+git commit -m "<YOUR COMMIT MESSAGE>"
+```
+
+The commit message should be short, concise and to the point. It should be written in the present tense (and not the past), and should describe what you did. For example, following the previous example
+
+```
+git commit -m "Add new team page"
+```
+
+Now it is time to push your changes. If it is the first time you push this branch, run the following command
+
+```
+git push --set-upstream origin <YOUR BRANCH NAME>
+```
+
+This will create your branch on the remote and push the changes to it. After that, if you want to push again you can run just a
+
+```
+git push
+```
+
+## Opening a Pull Request
+
+Remember when we said you could not push to `main` directly and how we would explain how to get your code into `main` later? This is the section. When you finish your work, you should open a Pull Request (or PR, for short), indicating you want to have your code merged into the `main` branch. To do that, go to GitHub, click the Pull Requests tab, click New Pull Request. Now you have to make multiple choices:
+
+- Base branch: `main`
+- Compare: your branch
+
+Click "Create New Pull Request". Now you must:
+
+- Name your PR: pick a name (in the present tense) that is short and descriptive
+- Pick some reviewers: try to pick people from the board on the current year, or people who have recently contributed to the same repository
+- Assign: always assign **yourself** to your PRs
```

Comment:
In case they aren't part of the organization , i don't think they can ask for a review or assign the PR on themselves